fennel/test_lib/mock_client.py

```diff
@@ -524,9 +541,31 @@ def extract_features(
             extractors, input_dataframe, output_feature_list, timestamps
         )
 
+    def extract_historical(
```

Comment:
Same feedback, the actual logic should be with the new API, when folks refer to it, and we will just kill the older API at some point

Reply:
done.